### PR TITLE
Expose the vars of somoffice to be used in Tryton

### PR DESCRIPTION
**Pull request by danypr92**

### CODE REVIEW

File: inventory/host_vars/local.somconnexio.coop/somoffice.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+somoffice_password: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          38643138643763303530633639376138363835323365353131656331383637626330616136303236
+          3136613231323732366239373337656532613264366466300a313966313135616266316539376566
+          39383132656331353963653436616532333235636461383835303135636362373636363838633031
+          6161383833303236300a646632653430303033346466363033626463616462646263646231333638
+          39613463663462373332656635643138333738353565366133313766363730653261393638393738
+          30646661626166366665376162643734663938663033663035643064383232633563343366356138
+          626238316331393239636566353166383431
+somoffice_user_creationg_enabled: 'True'
```
Comment (sauloperez) [inline inventory/host_vars/local.somconnexio.coop/somoffice.yml:13]:
```suggestion
somoffice_user_creation_enabled: 'True'
```

File: inventory/host_vars/stagingtryton.coopdevs.org/somoffice.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+somoffice_password: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          38643138643763303530633639376138363835323365353131656331383637626330616136303236
+          3136613231323732366239373337656532613264366466300a313966313135616266316539376566
+          39383132656331353963653436616532333235636461383835303135636362373636363838633031
+          6161383833303236300a646632653430303033346466363033626463616462646263646231333638
+          39613463663462373332656635643138333738353565366133313766363730653261393638393738
+          30646661626166366665376162643734663938663033663035643064383232633563343366356138
+          626238316331393239636566353166383431
+somoffice_user_creationg_enabled: 'True'
```
Comment (sauloperez) [inline inventory/host_vars/stagingtryton.coopdevs.org/somoffice.yml:13]:
```suggestion
somoffice_user_creation_enabled: 'True'
```

File: roles/systemd_service/templates/defaults_trytond.j2 [hunk 3]
```diff
@@ -38,3 +38,9 @@ BUGSNAG_RELEASE_STAGE="{{ bugsnag_release_stage }}"
 BUGSNAG_SOM_CONNEXIO_TRYTON_MODULE_API_KEY="{{ bugsnag_som_connexio_tryton_module_api_key }}"
 BUGSNAG_OPENCELL_SOMCONNEXIO_TRYTON_API_KEY="{{ bugsnag_opencell_somconnexio_tryton_api_key }}"
 BUGSNAG_CONTRACT_CHANGES_WIZARDS_SOMCONNEXIO_API_KEY="{{ bugsnag_contract_changes_wizards_somconnexio_api_key }}"
+
+# SomOffice vars
+SOMOFFICE_URL="{{ somoffice_url }}"
+SOMOFFICE_USER="{{ somoffice_user }}"
+SOMOFFICE_PASSWORD="{{ somoffice_password }}"
+SOMOFFICE_USER_CREATION_ENABLED="{{ somoffice_user_creationg_enabled }}"
```
Comment (sauloperez) [inline roles/systemd_service/templates/defaults_trytond.j2:46]:
```suggestion
SOMOFFICE_USER_CREATION_ENABLED="{{ somoffice_user_creation_enabled }}"
```

File: inventory/host_vars/local.somconnexio.coop/somoffice.yml (hunk 1 above)
Comment (danypr92) [reply inventory/host_vars/local.somconnexio.coop/somoffice.yml]: Done!

**Pull request merged**